db/migrate/20191223091535_populate_edition_editors.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class PopulateEditionEditors < ActiveRecord::Migration[6.0]
+  def change
+    revision_editors = Revision.joins(:editions_revisions)
+                       .group(:edition_id)
+                       .pluck(:edition_id, Arel.sql("ARRAY_AGG(DISTINCT created_by_id)"))
+                       .to_h
+
+    status_editors = Status.group(:edition_id)
+                     .pluck(:edition_id, Arel.sql("ARRAY_AGG(DISTINCT created_by_id)"))
+                     .to_h
+
+    Edition.all.each do |edition|
+      editor_ids = (revision_editors[edition.id] + status_editors[edition.id]).uniq
+      edition.edition_editor_ids = editor_ids
+    end
```

Comment:
We'll probably have to delete or amend this migration now it has run as soon as we removed the edition_editors association from Edition it will raise errors.